src/main/java/net/neoforged/neoforge/capabilities/color/IDyeColorable.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.capabilities.color;
+
+import net.minecraft.world.item.DyeColor;
+
+/**
+ * Represents a dye-colorable object in-game.
+ */
+public interface IDyeColorable {
```

Comment:
Same as IColorable, I'd like a `getColor()` though this one would likely return a `DyeColor`